catboost/libs/metrics/metric.h

```diff
@@ -290,6 +290,49 @@ THolder<IMetric> MakeUserDefinedQuerywiseMetric(const TMap<TString, TString>& pa
 THolder<IMetric> MakeAverageGainMetric(float topSize);
 
 
+//Normalized Gini metric
+struct TNormalizedGiniMetric: public TNonAdditiveMetric {
```

Comment:
This struct and `TNormalizedGiniMetric` must be moved to `.cpp` file and put into an anonymous namespace.

End user should not have access to solid class.